doc/user/content/sql/create-source/kafka.md

```diff
@@ -138,9 +138,10 @@ Note that:
 
 - The `DEBEZIUM` envelope is incompatible with this option.
 
-#### Partition, offset, timestamp
+#### Partition, offset, timestamp, headers
```

Comment:
Is this all the documentation there is on header metadata? Sigh. I'm on mobile for a bit, but will have a look later to improve it if it is (not blocking for the notes, though the anchor link might change).

Reply:
Wanna pull this out? I have a PR on the side for the headers docs.

Reply:
https://github.com/MaterializeInc/materialize/pull/11768

Reply:
Will do! I'm holding off on merging this until tomorrow afternoon, so I'll just rebase after you merge #11768 and update the link appropriately.

Reply:
Updated as promised!